File: app/src/main/java/dev/leonlatsch/olivia/util/Base64.java
package dev.leonlatsch.olivia.util;

/**
 * @author Leon Latsch
 * @since 1.0.0
 */
public class Base64 {

    public static String toBase64(byte[] bytes) {
        return android.util.Base64.encodeToString(bytes, android.util.Base64.NO_WRAP);
    }

    public static byte[] toBytes(String base64) {
        return android.util.Base64.decode(base64, android.util.Base64.NO_WRAP);
    }
}

File: app/src/main/java/dev/leonlatsch/kolibri/util/Base64.kt
package dev.leonlatsch.kolibri.util

/**
 * @author Leon Latsch
 * @since 1.0.0
 */
object Base64 {

    fun toBase64(bytes: ByteArray): String {
        return android.util.Base64.encodeToString(bytes, android.util.Base64.NO_WRAP)
    }

    fun toBytes(base64: String): ByteArray {
        return android.util.Base64.decode(base64, android.util.Base64.NO_WRAP)
    }
}
